features/support/env.rb

```diff
@@ -17,33 +17,22 @@
 Capybara.app_host = ENV["GOVUK_WEBSITE_ROOT"]
 phantomjs_logger = File.open("log/phantomjs.log", "a")
 
-BLACKLISTED_URLS = ['www.google-analytics.com']
+GOOGLE_ANALYTICS_URL = 'www.google-analytics.com'
+BLACKLISTED_URLS = [GOOGLE_ANALYTICS_URL]
 
 Capybara.register_driver :poltergeist do |app|
-  Capybara::Poltergeist::Driver.new(app, phantomjs_logger: phantomjs_logger)
+  options = {
+    debug: ENV['POLTERGEIST_DEBUG'] || false,
+    phantomjs_logger: phantomjs_logger,
+    url_blacklist: BLACKLISTED_URLS
```

Comment:
It's not quite the same as the original implementation (from PR #245), although it does look quite similar. I'm quite confident that this branch retains the behaviour of the fixed version (PR #260) and so we shouldn't see a repeat of issue #256.

Having said all that, you've made me want to double check the behaviour I'm seeing so that I can be _completely_ confident that I haven't reintroduced the problem described in issue #256.